src/components/measureGroups/MeasureObservationDetails.tsx

```diff
@@ -69,75 +70,103 @@ const MeasureObservationDetails = ({
             Remove
           </DSLink>
         )}
-        <Select
-          placeHolder={{ name: "-", value: "" }}
-          required={required}
-          readOnly={!canEdit}
-          label={label ? label : "Observation"}
-          id={`measure-observation-${name}`}
-          data-testid={`select-measure-observation-${name}`}
-          inputProps={{
-            "data-testid": `measure-observation-${name}-input`,
-          }}
-          style={{ width: 300 }}
-          value={measureObservation?.definition || ""}
-          onChange={(e) => {
-            if (onChange) {
-              onChange({
-                ...measureObservation,
-                definition: e.target.value,
-              });
-            }
-          }}
-          options={[
-            !required && (
-              <MenuItem key="-" value="">
-                -
-              </MenuItem>
-            ),
-            ...cqlFunctionNames.map((o) => (
-              <MenuItem key={o} value={o}>
-                {o}
-              </MenuItem>
-            )),
-          ]}
-        />
+        {canEdit && (
+          <Select
+            placeHolder={{ name: "-", value: "" }}
+            required={required}
+            disabled={!canEdit}
```

Comment:
I think this is no longer needed since it won't render unless canEdit is true. So..this will never be disabled when it's rendered.